doc/api/intl.md

```diff
@@ -38,26 +38,25 @@ in [BUILDING.md][].
 
```

Comment:
Earlier in this file there's a section:

> Node.js (and its underlying V8 engine) uses ICU to implement these features in native C/C++ code. <ins>However, some of them require a very large ICU data file in order to support all locales of the world. Because it is expected that most Node.js users will make use of only a small portion of ICU functionality, only a subset of the full ICU data set is provided by Node.js by default. Several options are provided for customizing and expanding the ICU data set either when building or running Node.js.</ins>

The underlined portion should probably be edited.

Reply:
- [ ] will do

Reply:
Basically, you could still build ICU with English only, but you'd have to do your own ICU build to get it. Such tools should live in ICU-land, not in Node.